README.md

```diff
@@ -1,229 +1,144 @@
 
-> wso2server.sh   (in UNIX)
 
-By default mode, it would ask you to enter the master password 
-(By default, master password is the password of carbon keystore and private key) 
+Join our community!
+========================
+- Read our documentation at [https://is.docs.wso2.com/en/latest/](https://is.docs.wso2.com/en/latest/).
+- Get help on [Stack Overflow](https://stackoverflow.com/questions/tagged/wso2-is).
+- Join the conversation on [Discord](https://discord.gg/wso2).
+- Learn more by reading articles from our [library](https://wso2.com/library/).
 
-3. Change any password by running ciphertool script from bin directory.  
 
-> ciphertool -Dchange  (in UNIX)
 
-For more details see
-https://is.docs.wso2.com/en/7.1.0/deploy/security/encrypt-passwords-with-cipher-tool/
 
-Support
-=======
-We are committed to ensuring that your enterprise middleware deployment is completely supported from
-evaluation to production. Our unique approach ensures that all support leverages our open development
-methodology and is provided by the very same engineers who build the technology.
 
-For more details and to take advantage of this unique opportunity, visit http://wso2.com/support/.
+Commercial Support
+==================
+You can take advantage of a WSO2 on-prem product subscription for the full range of software product benefits like expert support, continuous product updates, vulnerability monitoring, and access to the licensed distribution for commercial use.
 
+To learn more, check WSO2 Subscription.
 
-For more information on WSO2 Carbon, visit the WSO2 Oxygen Tank (http://wso2.org)
 
-For more information about WSO2 Identity Server please see http://wso2.org/projects/identity or visit the
-WSO2 Oxygen Tank developer portal for addition resources.
 
-For further details, see the WSO2 Carbon documentation at
-https://docs.wso2.com/display/Carbon4411/WSO2+Carbon+Documentation
+Contributing
+===============
+If you are planning on contributing to the development efforts of WSO2 <Product Name>, you can do that by checking out the latest development version. The master branch holds the latest unreleased source code.
 
-Survey On Open Source Community Communication
-=============================================
+Please follow the details instructions available here: https://wso2.github.io  
 
+<If necessary, insert specific instructions for your product here>
+Can you fill this survey ?
 WSO2 wants to learn more about our open source software (OSS) community and your communication preferences to serve you better.
 
 In addition, we may reach out to a small number of respondents to ask additional questions and offer a small gift.
 
-Link to survey: https://forms.gle/h5q4M3K7vyXba3bK6
-
+Survey is available at: https://forms.gle/h5q4M3K7vyXba3bK6 
 
 ---------------------------------------------------------------------------
-(c) Copyright 2022 WSO2 LLC.
+(c) Copyright 2019 - 2025 WSO2 LLC. All Rights Reserved.
```

Comment:
Shall we remove the year so that we wouldn't have to update the readme every year?

Also, we don't normally use "All Rights Reserved" phrase on open source software components.

```suggestion
(c) Copyright WSO2 LLC.
```

Reply:
+1

Reply:
@pavinduLakshan again, shall we discuss in this doc as this would effect all the products? https://docs.google.com/document/d/1h2YHSNzLY7WeG-OFhiXWAmV3Ge6BYh5OxWzh1MSpDPE/edit?tab=t.0#heading=h.ets5v6jkucdz